### DISCUSSION TITLE
Support bun?

### DISCUSSION THREAD

**Discussion by L1lith** [withastro/roadmap#224] [Proposal]
See [bun](https://bun.sh/)

**Comment by natemoo-re** [comment]
Hey @L1lith! Bun is a project we'd LOVE to support. Last time I checked, `vite` still can't run in `bun`, but as soon as that's unblocked, we should be able to support Bun.

It's definitely something we're keeping a close eye on.

**Comment by ghost** [comment]
Hi @natemoo-re, it appears to be possible now https://bun.sh/guides/ecosystem/vite.

There are even third party adapters like https://github.com/nurodev/astro-bun, though I would prefer official support.

Please let me knows if there is anything else that's required for this RFC to proceed.

Thanks.
